docs/source/api/plugin/subscription-callback.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: "API Reference: Subscription callback plugin"
+api_reference: true
+minVersion: 4.9.0
+---
+
+> This protocol is currently in [preview](/resources/product-launch-stages#preview). Breaking changes might be introduced during the preview period.
+>
+> This document covers the usage of the Subscription callback plugin for use in Federation with Apollo Router. For more information about the protocol itself, see the [subscription callback protocol](/router/executing-operations/subscription-callback-protocol).
```

Comment:
```suggestion
> This document covers the usage of the Subscription callback plugin for use in Apollo Federation with Apollo Router. For more information about the protocol itself, see the [subscription callback protocol](/router/executing-operations/subscription-callback-protocol).

**This protocol is currently in [preview](/resources/product-launch-stages#preview). Breaking changes might be introduced during the preview period.**
```

According to our "official verbiage", the first instance of "federation" should be "Apollo Federation", with subsequent mentions with lowercase "federation".